src/cbc_sdk/platform/policy_ruleconfigs.py

```diff
@@ -402,21 +395,14 @@ def _set(self, attrname, new_value):
             if self._parent and pristine and attrname in self._dirty_attributes:
                 self._parent._mark_changed()
 
-        def _flatten(self):
-            """
-            Turns this rule into a dict for transferral to the server.
-
-            Returns:
-                dict: The information defining the rule.
-            """
-            return copy.deepcopy(self._info)
-
         def remove(self):
             """Removes this rule from the rule group that contains it."""
             if self._parent:
                 group_list = [group for group in self._parent.rule_groups if self in group._rules]
                 if group_list:
+                    location = [ndx for ndx, item in enumerate(group_list[0]._rules) if item == self]
```

Comment:
Have you considered using `list.index()` instead of the custom one liner loop into a list. The index function does suppport object values as the input. Python reference: https://docs.python.org/3/tutorial/datastructures.html

Reply:
This will also be safer as you may get an index out of range error if the object is not found

Reply:
OK, I'm rewriting this to use `index()` instead. In this context, it will _not_ fail, though, since we already know `self` exists in `group_list[0].rules`, because that was why the group got selected in the search expression that created `group_list`.